File: cryptography/hash.go
package main

import (
	"crypto/sha256"
	"encoding/hex"
	"fmt"
)

// Función para hashear con SHA-256
func hashSHA256(data string) string {
	hash := sha256.Sum256([]byte(data))
	return hex.EncodeToString(hash[:])
}

func main() {
	text := "Hola, mundo! 1234"
	hashedText := hashSHA256(text)

	fmt.Println("Texto original:", text)
	fmt.Println("Hash SHA-256:", hashedText)
}

File: cryptography/node/hash.js
const crypto = require('crypto');

// Función para hashear con SHA-256
function hashSHA256(data) {
    return crypto.createHash('sha256').update(data).digest('hex');
}

const text = "Hola, mundo! 1234";
const hashedText = hashSHA256(text);

console.log("Texto original:", text);
console.log("Hash SHA-256:", hashedText);
